### ISSUE TITLE
Doesn't work :(

### ISSUE THREAD

**Issue by Ragnell13** [illuminatedwax/pesterchum#82]
So I have the newest 3.41 Pesterchum version (The one you download from the thread when installing Pesterchum from zero on a new computer).

I start it up; works normally for a while. But when I get into a log with someone or into a Memo suddenly I stop receiving replies from the conversations. My pestermates tell me (Via Skype) I go offline and exit the memo/log, but from my POV everything looks okay (Save no incoming replies). No error messages show up, I can still start new conversations (Still doesn't get me a reply) and use Pesterchum normally but to other users I am not there.

And if I try to do anything to try to fix it it tells me my handle is being used already and forces me to choose another handle, even though I have only ONE pesterchum open.

Simply put it's not working correctly. I can't upload images cause from my view things are working fine AND the only error message I receive is the one telling me I have multiple handles open. Any ideas on how to fix it? Or if not, do you know where I can get the older version? That one worked just fine.
